backend/src/main/resources/application.yaml

```diff
@@ -44,6 +44,9 @@ okta:
     org-url: https://hhs-prime.okta.com
     token: ${OKTA_API_KEY:MISSING}
 simple-report:
+  address-validation:
+    smarty-auth-id: ${SMARTY_AUTH_ID:MISSING}
+    smarty-auth-token: ${SMARTY_AUTH_TOKEN:MISSING}
```

Comment:
1. Ugh I hate placeholders—can we just use an `Optional` if we want to check that?
2. Suggested change: `client-id` and `auth-token` or `token`.